x-pack/metricbeat/module/prometheus/remote_write/data.go

```diff
@@ -44,9 +44,17 @@ func remoteWriteEventsGeneratorFactory(base mb.BaseMetricSet) (remote_write.Remo
 	}
 
 	if config.UseTypes {
+
+		// /metricbeat/module/mb/mb.go defines default as Period:  time.Second * 10,
+		// We are setting Period at least 60secs because of issue https://github.com/elastic/beats/issues/38458
```

Comment:
```suggestion
		by default prometheus push data with the interval 60s, in order to calculate counter rate we are setting Period to 60secs accordingly
```
maybe provide short summary instead of the all discussion

Reply:
Done in [cbc2d4c](https://github.com/elastic/beats/pull/38553/commits/cbc2d4c6e8953c5e2c5ce55d9e5947d5f6971f8f)